core/src/main/java/org/apache/iceberg/Partitioning.java

```diff
@@ -215,11 +225,12 @@ public Void alwaysNull(int fieldId, String sourceName, int sourceId) {
    * that have the same field ID but use a void transform under the hood. Such fields cannot be part
    * of the grouping key as void transforms always return null.
    *
+   * @param schema a schema
```

Comment:
I think this needs a little more description as well as information about what "null" means in this context. The Schema here is the projected schema to pull transforms from?

Reply:
Will add.

Reply:
Done. Resolving.